packages/workflows-shared/src/engine.ts

```diff
@@ -142,6 +142,11 @@ export class Engine extends DurableObject<Env> {
 		_accountId: number,
 		_instanceId: string
 	): Promise<InstanceStatus> {
+		// NOTE(dferreira): we need to verify if the engine was initialized or not
```

Comment:
are you tracking this anywhere? if yes, we can remove the comment 🙏

Reply:
It was more of an explanation of why it was necessary. Removing it now.